hibernation-setup-tool.c

```diff
@@ -979,11 +982,6 @@ static void perform_fs_specific_checks(const char *path)
             try_spawn_and_wait("btrfs", 3, "filesystem", "defragment", path);
         return;
     }
-
```

Comment:
May be consider leaving the code around & commented out.. and with the above comment mentioning why we don't enable this. And, if needed we enable/flight this later this via some flag/arg.